spectral-octal-number-issue.md

```diff
@@ -0,0 +1,109 @@
+# Non-Octal Numbers In Unquoted Strings
+
+## Issue Description
+
+When linting OpenAPI 3.x.x specifications, Spectral lint is not appropriately
+parsing strings per the YAML specification. This issue presents itself when
```

Comment:
"This issue seems to present itself..."?

Reply:
Updating.